{"name": "Создание двух курьеров с одинаковым логином", "status": "passed", "description": "Проверяем, что при создании двух курьеров с одинаковым логином возвращается ошибка", "start": 1721642991584, "stop": 1721642991930, "uuid": "1ea7116e-c31e-457b-a57b-886657a2ae24", "historyId": "a79749f11be2dd76e09d56b94d635323", "testCaseId": "a79749f11be2dd76e09d56b94d635323", "fullName": "tests.test_create_courier.TestCreateCourier#test_create_courier_with_same_login", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_courier"}, {"name": "subSuite", "value": "TestCreateCourier"}, {"name": "host", "value": "MacBook-Pro-4.local"}, {"name": "thread", "value": "66649-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_courier"}]}